backend/src/main/java/de/daniellohrey/betterist/controller/ProductController.java

```diff
@@ -36,7 +37,7 @@ public Product findProductById(@PathVariable String id) {
     }
 
     @PostMapping
-    public Product addProduct(@RequestBody ProductDto productDto) {
+    public Product postProduct(@RequestBody ProductDto productDto) {
```

Comment:
you may want to consider naming your methods after what they do and not what request type they use as it is convention and helps others to read and understand your code